src/com/ableton/PythonPackageVersionReader.groovy

```diff
@@ -0,0 +1,56 @@
+package com.ableton
+
+
+/**
+ * Class to get the version of a Python package by parsing the file that sets it.
+ */
+class PythonPackageVersionReader implements Serializable {
+  /**
+   * Script context.
+   * <strong>Required value, may not be null!</strong>
+   */
+  @SuppressWarnings('FieldTypeRequired')
+  def script = null
+
+  /**
+   * Start of the line setting the Python package version.
+   */
+  String versionLineBeginning = "__version__ = '"
```

Comment:
I'm not sure I like `'` being included in this string, and for that matter, the presence of the `versionLineEnding` operator below. Might it make more sense to just define this as `versionLineBeginning = '__version__'` and then have `boolean stringsAreSingleQuoted = true`? Then `=` is also somewhat assumed.